lib/cc/cli/validate_config.rb

```diff
@@ -48,28 +50,47 @@ def errors
       def display_issues
         display_errors
         display_warnings
+        validate_engines
         display_nested_warnings
       end
 
       def display_errors
         errors.each do |error|
-          puts colorize("ERROR: " + error, :red)
+          puts colorize("ERROR: #{error}", :red)
         end
       end
 
       def display_nested_warnings
         nested_warnings.each do |nested_warning|
           if nested_warning[0][0]
-            puts colorize("WARNING in " + nested_warning[0][0] + ": " + nested_warning[1], :red)
+            puts colorize("WARNING in #{nested_warning[0][0]}: #{nested_warning[1]}", :red)
           end
         end
       end
 
       def display_warnings
         warnings.each do |warning|
-          puts colorize("WARNING: " + warning, :red)
+          puts colorize("WARNING: #{warning}", :red)
         end
       end
+
+      def validate_engines
```

Comment:
What do you think of naming this `display_invalid_engines` to match the convention elsewhere?